src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+interface Props {
+  filteredPeople: Person[];
+  setSelectedPeople: (person: Person) => void;
+}
+
+export const PersonList: React.FC<Props> = React.memo(
+  ({ filteredPeople, setSelectedPeople }) => {
+    return (
+      <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+        <div className="dropdown-content">
+          {filteredPeople.map((person: Person) => (
+            <div
+              className="dropdown-item"
+              data-cy="suggestion-item"
+              role="button"
+              tabIndex={0}
+              key={person.slug}
+              onClick={() => setSelectedPeople(person)}
+              onKeyDown={(e) => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                  setSelectedPeople(person);
+                }
+              }}
```

Comment:
Add handlers here to improve readability and code style. It will be a good practice